mmdet3d/ops/iou3d/iou3d_utils.py

```diff
@@ -4,13 +4,15 @@
 
 
 def boxes_iou_bev(boxes_a, boxes_b):
-    """
-    :param boxes_a: (M, 5)
-    :param boxes_b: (N, 5)
-    :return:
-        ans_iou: (M, N)
-    """
+    """Calculate boxes IoU in the bird view.
+
+    Args:
+        boxes_a (torch.Tensor): Input boxes a with shape (M, 5).
+        boxes_b (torch.Tensor): Input boxes b with shape (N, 5).
 
+    Returns:
+        ans_iou (torch.Tensor): IoU result with shape (M, N).
+    """
```

Comment:
1. We may also fix this API by setting device.
2. ans_iou = boxes_a.new_zeros(torch.Size((boxes_a.shape[0], boxes_b.shape[0])))